jaxley/modules/cell.py

```diff
@@ -85,28 +86,35 @@ def __init__(
             # self.xyzr at `.vis()`.
             self.xyzr = [float("NaN") * np.zeros((2, 4)) for _ in range(len(parents))]
 
-        self.nseg_per_branch = jnp.asarray([branch.nseg for branch in self.branch_list])
+        self.total_nbranches = len(branch_list)
+        self.nbranches_per_cell = [len(branch_list)]
+        self.comb_parents = jnp.asarray(parents)
+        self.comb_children = compute_children_indices(self.comb_parents)
+        self.cumsum_nbranches = jnp.asarray([0, len(branch_list)])
+
+        # Compartment structure. These arguments have to be rebuilt when `.set_ncomp()`
+        # is run.
+        self.nseg_per_branch = jnp.asarray([branch.nseg for branch in branch_list])
         self.nseg = int(jnp.max(self.nseg_per_branch))
```

Comment:
should we rename this to `max_nseg`? or make this an array?

Reply:
Yes I agree but let's do this in another PR.